apps/landingPage/containers/LandingPage.js

```diff
@@ -28,6 +30,8 @@ export default class LandingPage extends React.Component<{}, State> {
   };
 
   componentDidMount() {
+    ReactGA.initialize(GOOGLE_ANALYTICS_TRACKING_ID);
```

Comment:
What about adding condition here to avoid tracking localhost? `document.location.hostname != 'localhost'` ?

Reply:
OK, done

Reply:
Thanks